transportable-udfs-trino-plugin/src/main/java/com/linkedin/transport/trino/TransportConnectorMetadata.java

```diff
@@ -37,12 +42,23 @@ public FunctionDependencyDeclaration getFunctionDependencies(ConnectorSession se
   @Override
   public Collection<FunctionMetadata> getFunctions(ConnectorSession session, SchemaFunctionName name) {
     return functions.values().stream().map(StdUdfWrapper::getFunctionMetadata)
-        .filter(e -> e.getCanonicalName().equals(name.getFunctionName()))
+        // Function are expected to be called case agnostic
+        .filter(e -> e.getCanonicalName().toLowerCase(ENGLISH).equals(name.getFunctionName()))
```

Comment:
do we need to lower-case `name.getFunctionName()` as well?

Reply:
`name` here is expected to be in all lower case. Trino parses `Array_Element_At` to `QualifiedName` and cast the original parts to lower case, which is then used by `MetadataManager#toQualifiedFunctionName`.